cmake/Hunter/config.cmake

```diff
@@ -18,8 +18,3 @@ hunter_config(
 )
 
 hunter_config(Boost VERSION 1.65.1)
-
```

Comment:
Naive question - how does removing the ethash hunter_config section result in us using 0.4.4? Does it result in us using the latest version? If so, should we lock our config to a particular version so upgrades are intentional?

Reply:
It uses Hunter's default version which is 0.4.4 (I added it yesterday).

Reply:
Should we include an explicit Ethash version so we don’t get unintentional upgrades? I know that it ultimately probably doesn’t matter since we are presumably the only people publishing new ethash packages, but i don’t think it would be a bad idea from a “best practices” standpoint.